classes/Errors/DefaultErrorHandler.php

```diff
@@ -43,37 +43,37 @@ public function __construct($debug)
 	/**
 	 * {@inheritdoc}
 	 */
-	public function handle(Exception $exception)
+	public function handle(Throwable $throwable)
 	{
 		if ($this->debug && class_exists('Whoops\Run')) {
-			return $this->handleWithWhoops($exception);
+			return $this->handleWithWhoops($throwable);
 		}
 
-		return $this->handleWithSymfony($exception);
+		return $this->handleWithSymfony($throwable);
 	}
 
-	protected function handleWithWhoops(Exception $exception)
+	protected function handleWithWhoops(Throwable $throwable)
 	{
 		$whoops = new Run();
 		$whoops->allowQuit(false);
 		$whoops->writeToOutput(false);
 		$whoops->pushHandler(new PrettyPageHandler());
 
-		return $whoops->handleException($exception);
+		return $whoops->handleException($throwable);
 	}
 
-	protected function handleWithSymfony(Exception $exception)
+	protected function handleWithSymfony(Throwable $throwable)
 	{
-		if (!$exception instanceof FlattenException) {
-			$exception = FlattenException::create($exception);
+		if (!$throwable instanceof FlattenException) {
+			$throwable = FlattenException::createFromThrowable($throwable);
 		}
 
 		$handler = new ExceptionHandler($this->debug);
 
 		return Response::create(
-			$handler->getHtml($exception),
-			$exception->getStatusCode(),
-			$exception->getHeaders()
+			$handler->getHtml($throwable),
+			$throwable->getStatusCode(),
```

Comment:
I am concerned about this line. Throwable doesn't have a status code.

https://www.php.net/manual/en/class.throwable.php .

This is true for exception also. So may be this is not necessary itself ?

Reply:
`$throwable` is an instance of symfony's `FlattenException` at this point, which does have these methods.

Reply:
Cool